packages/stream-metadata/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM node:lts-alpine3.20 as builder
+
+WORKDIR /river
+
+# monorepo root config
+COPY ./package.json ./package.json
+COPY ./.yarn/plugins ./.yarn/plugins
+COPY ./.yarn/releases ./.yarn/releases
+COPY ./remappings.txt ./remappings.txt
+COPY ./.yarnrc.yml ./.yarnrc.yml
+COPY ./buf.yaml ./buf.yaml
+COPY ./lerna.json ./lerna.json
+COPY ./yarn.lock ./yarn.lock
+COPY ./turbo.json ./turbo.json
+COPY ./packages/tsconfig.base.json ./packages/tsconfig.base.json
+
+# monorepo core dependencies
+COPY ./protocol ./protocol
+
+# monorepo scripts
+COPY ./scripts ./scripts
+
+# monorepo packages
+COPY ./packages/eslint-config /river/packages/eslint-config
+COPY ./packages/prettier-config /river/packages/prettier-config
+COPY ./packages/generated /river/packages/generated
+COPY ./packages/web3 /river/packages/web3
+COPY ./packages/dlog /river/packages/dlog
+COPY ./packages/proto /river/packages/proto
+COPY ./packages/sdk /river/packages/sdk
+COPY ./packages/encryption /river/packages/encryption
+COPY ./packages/stream-metadata /river/packages/stream-metadata
+
+# install dependencies and build
+RUN apk add --no-cache git curl
+RUN corepack enable
+RUN yarn install
+RUN yarn run turbo build --filter @river-build/stream-metadata
+
+# create runner image with only the necessary files
+FROM node:lts-alpine3.20 as runner
+COPY --from=builder /river/packages/stream-metadata/dist /river/packages/stream-metadata/dist
+
+# run the service
+WORKDIR /river/packages/stream-metadata
+ENV NODE_ENV=production 
+CMD ["node", "./dist/node_esbuild.cjs"]
```

Comment:
Need to pass --enable-source-map to get it to actually pickup the sourcemaps. Not sure why, as the docs say it should be enabled by default, but my experience is the flag is required